smart_proxy_discovery.gemspec

```diff
@@ -16,9 +16,9 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ['README.md', 'LICENSE']
   s.files = Dir['{lib,settings.d,bundler.d}/**/*'] + s.extra_rdoc_files
   s.homepage = 'https://github.com/theforeman/smart_proxy_discovery'
-  s.license = 'GPLv3'
+  s.license = 'GPL-3.0-only'
```

Comment:
Small remark: You specify the licence string to GPL-3.0-only but the LICENCE file is for GPL-3.0-or-later.

Reply:
I always thought that the license text is identical, it's just how you apply it to your code?
(The old tag, `GPLv3` also means `only`, so this PR at least doesn't change any meaning)

Reply:
Just search for 'later' in `LICENCE`. I'm no expert in this and this is probably not the scope of this change. But out of curiosity, where do you take it from that `GPLv3` implies `only`?

Reply:
> Just search for 'later' in `LICENCE`.

I did:
```
  Each version is given a distinguishing version number.  If the
Program specifies that a certain numbered version of the GNU General
Public License "or any later version" applies to it, you have the
option of following the terms and conditions either of that numbered
version or of any later version published by the Free Software
Foundation.  If the Program does not specify a version number of the
GNU General Public License, you may choose any version ever published
by the Free Software Foundation.
```

However, the program does *not* specify that.

> But out of curiosity, where do you take it from that `GPLv3` implies `only`?

https://guides.rubygems.org/specification-reference/#license= points to https://spdx.org/licenses/ for the recommended ways of noting down licenses.

https://spdx.org/licenses/#deprecated has a `GPL-3.0` identifier, which is to be interpreted as `only`.

Granted, it's not saying `GPLv3` anywhere. For that we could use https://docs.fedoraproject.org/en-US/legal/allowed-licenses/, which leads to the same interpretation: "only" (or later would be `GPLv3+`).

Reply:
You're right, I missed the "If the Program specifies".

Reply:
And thanks for the clarification!

Reply:
:)

The "problem" is that the license text is set in stone (if you'd change it, you'd create a new version of the license…)

Individual programs are supposed to document their intent (see https://github.com/theforeman/smart_proxy_discovery/blob/develop/LICENSE#L624) but the format of that is not mandatory.

And the rest is then up to interpretation by lawyers (which we both luckily are not ;) )

Reply:
So we can conclude this is all valid and we can merge it?

Reply:
That's my reading, yes

Reply:
Yes, you can.